inbox/api/jsc_api.py

```diff
@@ -118,3 +120,32 @@ def auth_callback():
         })
 
         return make_response( (resp, 201, { 'Content-Type': 'application/json' }))
+
+@app.route('/retry_sync', methods=['POST'])
+def retry_sync():
+    g.parser.add_argument('namespace_public_id', required=True, type=valid_public_id, location='form')
+
+    args = strict_parse_args(g.parser, request.args)
+    shard = (args.get('target') or 0) >> 48
```

Comment:
I think you missed the `add_argument` for `target`, otherwise this will always be `None` (and fallback to `0`)